System/Win32/Types.hs

```diff
@@ -26,10 +26,10 @@ import Data.Char (isSpace)
 import Data.Int (Int32, Int64)
 import Data.Maybe (fromMaybe)
 import Data.Word (Word8, Word16, Word32, Word64)
-import Foreign.C.Error (getErrno, errnoToIOError)
+import Foreign.C.Error (Errno(..), errnoToIOError)
```

Comment:
GHC 8.0.1 then gives an error:
```
`Errno' cannot be marshalled in a foreign call
  because its data constructor is not in scope
  Possible fix: import the data constructor to bring it into scope
```

GHC 7.8.4 simply says `Unacceptable result type in foreign declaration: IO Errno`. I believe this is related: https://ghc.haskell.org/trac/ghc/ticket/5610 (GHC started requiring `newtype` constructors to be in scope for FFI).